aliparadise/Dose
# Added list of Patient Medications by Patient Id

closes #3  
Changes Made

1. Added file `PatientMedication.cs` to `Models` directory.
2. Added file `PatientMedicationRepository.cs` and `IPatientMedicationRepository.cs` to `Repositories` directory.
3. Added file `ManageMeds.cshtml` to Patient Views.
1. Modified file `PatientMedicationRepostory.cs` to include `GetAllPatientMedicationsByPatientId` and functionality.
2. Modified file `PatientController.cs` to include `GetAllPatientMedicationsByPatientId` and functionality.
   

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout ap-patientMedication
   ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
   > When an authorized user is logged in and is viewing the Patient Index, the user should be able to click on the "Manage Meds" link for a patient, and see the list of medications that specific patient is using.
